test/test_board.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from constans.constans import (
+    PLAYER_1,
+    PLAYER_2,
+    INITIAL_POSITION_PLAYER_1,
+    INITIAL_POSITION_PLAYER_2
+    )
+from game.board import Board
+from game.player import Player
+
+from constans.constants_game import (
+    LARGE,
+)
```

Comment:
- Remove blank space
- Wrong order
- One line import
```suggestion
from constans.constants_game import LARGE
from game.board import Board
from game.player import Player
```